modules/jaeger-rn-fixed-issues.adoc

```diff
@@ -0,0 +1,22 @@
+////
+Module included in the following assemblies:
+- jaeger-release-notes.adoc
+////
+
+[id="jaeger-rn-fixed-issues_{context}"]
+= Jaeger Fixed issues
+////
+Provide the following info for each issue if possible:
+Consequence - What user action or situation would make this problem appear  (If you have the foo option enabled and did x)? What did the customer experience as a result of the issue? What was the symptom?
+Cause - Why did this happen?
+Fix - What did we change to fix the problem?
+Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
+////
+
+BZ# https://bugzilla.redhat.com/show_bug.cgi?id=000001[000001]
+
+There was a werewolf in the last release.  You would have only encountered this werewolf if it was a full moon and you were outside.  The werewolf has been shot with a silver bullet and no longer exists.
+
```

Comment:
I don't see any fixed issues nor any witty comments about werewolves or vampires when reviewing this page: https://boj_6--ocpdocs.netlify.com/openshift-enterprise/latest/jaeger/rhbjaeger-release-notes.html#jaeger-rn-new-features_jaeger-release-notes  Am I looking at the right thing?

Reply:
Since it's our first release, we're not publishing any fixed issues.  (My boilerplate werewolf and vampire examples will be replaced with actual content in the next release. )